contribute/CONTRIBUTING-TO-DEVELOPER-DOC.md

```diff
@@ -0,0 +1,55 @@
+# Contributing to OpenEBS Developer Documentation
+
+This document describes the process for improving developer documentation by creating PRs at OpenEBS.
+
+The developer documentation includes anything that will help the community like:
+
+- Architecture and Design documentation
+- Technical or Research Notes
+- FAQ
+- Process Documentation
+- Generic / Miscellaneous Notes
+
+We are also looking for an explainer video or presentation from the community that helps new developers in understanding the OpenEBS Architecture, it's use cases, and installation procedure locally.
+
+At a very high level, the process to contributing and improving the code is pretty simple:
+
+- Submit an issue describing your proposed change
+- Create your development branch
+- Commit your changes
+- Submit your Pull Request
+
+The following sections describe some guidelines that can come in handy with the above process.
+*Followed by the guidelines, is a [cheatsheet](https://github.com/openebs/openebs/blob/master/contribute/git-cheatsheet.md) with frequently used git commands.*
+
+## Submit an issue describing your proposed change
+
+Some general guidelines when submitting issues for developer documentation:
+
+- If the proposed change requires an update to the existing page, please provide a link to the page in the issue.
+- If you want to add a new page, then go ahead and open an issue describing the requirement for the new page.
+
+You can also help with some existing issues under this category available at [developer documentation issues list](https://github.com/openebs/openebs/labels/documentation%2Fdevel)
```

Comment:
This label doesn't exist!
We do have a `Documentation` label, which I will rename to `kind/documentation`

Reply:
Done. Changed to `kind/documentation`